src/api/FetchWrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Advisory: It might be worth rethinking using a class here.

I have tried using them before and they can quickly get quite difficult to manage and they do not behave like traditional OO classes.

Reply:
Replaced with a function object in https://github.com/sanger/traction-ui/pull/1978/commits/2af437eec165b1484207ad35073aac11852a480d.